x-pack/plugin/rollup/src/test/java/org/elasticsearch/xpack/rollup/config/ConfigTests.java

```diff
@@ -19,32 +19,25 @@
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
 import static org.hamcrest.Matchers.equalTo;
-
+//TODO split this into dedicated unit test classes (one for each config object)
 public class ConfigTests extends ESTestCase {
 
     public void testEmptyField() {
-        MetricConfig.Builder config = ConfigTestHelpers.getMetricConfig();
-        config.setField(null);
-        Exception e = expectThrows(IllegalArgumentException.class, config::build);
-        assertThat(e.getMessage(), equalTo("Parameter [field] must be a non-null, non-empty string."));
-
-        config.setField("");
-        e = expectThrows(IllegalArgumentException.class, config::build);
-        assertThat(e.getMessage(), equalTo("Parameter [field] must be a non-null, non-empty string."));
+        final String field = randomBoolean() ? "" : null;
```

Comment:
Is there any advantage in randomizing these?  I assumed that since unit tests are fast to execute, we should go ahead and test the obvious paths all the time so that failures aren't flaky.

Reply:
You're right, Jim made the same remark on another pull request. I removed the randomization.